### ISSUE TITLE
Consider putting on elpa ?

### ISSUE THREAD

**Issue by olymk2** [steckerhalter/ob-php#3]
This no longer exists 

https://melpa.org/#/ob-php

I think this is because its based from this repository which has been archived.

https://github.com/stardiviner/ob-php

Could this be packaged in its placed ?

**Comment by steckerhalter** [comment]
It probably could, does it still work? I'm not using php anymore for quite a while. Another option would be if you'd use [quelpa](https://github.com/quelpa/quelpa) to install it. What do you think?

**Comment by olymk2** [comment]
I have not actually tried it, I will give it a try at some point from what I can tell quelpa does not have anything like list packages so you cant install directly in emacs ?

good to know about as this is the first time i have heard of quelpa, I like to avoid doing php but unfortunately legacy projects use it :/

**Comment by steckerhalter** [comment]
> from what I can tell quelpa does not have anything like list packages so you cant install directly in emacs ?

how do you mean "install directly in emacs"? you can install packages from MELPA with `M-x quelpa`. for anything else you need to provide a recipe

**Comment by olymk2** [comment]
that's basically what I mean there are various package managers in emacs for installing software, for example i call package-show-package-list or install-package and search for what i need then hit install and thats it so i don't need to go search online and put in custom lisp to pull from other sources like git or a wiki.

just convenience really.

**Comment by steckerhalter** [comment]
right. you don't get a list with details, but you do get a list with package names to choose from.

**Comment by olymk2** [comment]
package-show-package-list actually does give you a description and more when you hit enter on a package, quite handy :)

![image](https://user-images.githubusercontent.com/1034950/37205141-8dd2d68a-238b-11e8-8069-ff08c5e5d2c2.png)

**Comment by steckerhalter** [comment]
I meant quelpa doesn't give you the details :smiley:
